packages/manager/modules/pci/src/projects/project/storages/databases/add/add.controller.js

```diff
@@ -346,9 +352,10 @@ export default class {
         : this.model.flavor.hourlyPricePerGB
       ).priceInUcents || 0;
     return (
-      this.model.plan.nodesCount *
-      (flavorPrice.priceInUcents +
-        additionalDiskPrice * this.model.disk.additionalDiskSize)
+      this.model.plan.nodesCount * flavorPrice.priceInUcents +
+      additionalDiskPrice *
+        this.model.disk.additionalDiskSize *
+        this.storageNodeFactor
```

Comment:
**suggestion:**
```suggestion
      (this.model.plan.nodesCount * flavorPrice.priceInUcents) +
      (additionalDiskPrice *
        this.model.disk.additionalDiskSize *
        this.storageNodeFactor)
```

Reply:
I have added those parenthesis, but they are automatically removed by the linter.